.lift/config.toml

```diff
@@ -1,3 +1,8 @@
 jdk11 = true
-ignoreRules = [ "THREAD_SAFETY_VIOLATION", "MissingSummary", "SameNameButDifferent" ]
+ignoreRules = [ "THREAD_SAFETY_VIOLATION", "MissingSummary", "SameNameButDifferent", "NonAtomicVolatileUpdate", "EqualsGetClass"]
 setup = ".lift/setup.sh"
+ignoreFiles = """
+skywalking-ui/
+apm-protocol/apm-network/target/
```

Comment:
Could we ignore folders lik `**/generated-sources/**`? All protoc generated files are there, but in different folder paths.

Reply:
@wu-sheng Do you mean instead of the target directory or as well as?

Reply:
Makes sense. 
I've added generated sources https://github.com/apache/skywalking/pull/8769/commits/9902e290de934f211e88ffbd2d4e99bb2f01d4f5

For safety I've also excluded any target directory 
https://github.com/apache/skywalking/pull/8769/commits/5b398d558f19e3419bf7de79bdfade3330c46f21